File: src/main/java/com/acn/vinlotteri/model/Ticket.java
package com.acn.vinlotteri.model;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.*;
import org.springframework.data.annotation.CreatedDate;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.time.LocalDateTime;

@Entity
@Builder(toBuilder = true)
@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
@JsonInclude(JsonInclude.Include.NON_NULL)
public class Ticket {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;
    private long lotteryId;
    private String userName;
    private boolean winner;
    private String createdDate;

}

File: src/static/api/Ticket.php
<?php

class Ticket
{
    public $id;
    public $lotteryId;
    public $userName;
    public $winner;
    public $createdDate;

    public function __construct($id, $lotteryId, $userName, $winner, $createdDate)
    {
        $this->id = $id;
        $this->lotteryId = $lotteryId;
        $this->userName = $userName;
        $this->winner = $winner;
        $this->createdDate = $createdDate;
    }
}